pkg/scheduling/volumeusage.go

```diff
@@ -146,7 +143,7 @@ func resolveDriver(ctx context.Context, kubeClient client.Client, volumeName str
 		//  2. The StorageClass never existed and was used to bind the PVC to an existing PV, but that PV was removed
 		// In either of these cases, we should ignore the PVC while computing limits and continue.
 		if errors.IsNotFound(err) {
-			logging.FromContext(ctx).With("storageclass", storageClassName).Errorf("failed updating volume limits for volume with unbound PVC, %w", err)
+			logging.FromContext(ctx).With("pod", pod.Name).With("pvc", pvc.Name).With("storageclass", storageClassName).Errorf("failed updating volume limits for volume with unbound PVC, %w", err)
```

Comment:
```suggestion
			logging.FromContext(ctx).With("pod", pod.Name, "pvc", pvc.Name).With("storageclass", storageClassName).Errorf("failed updating volume limits for volume with unbound PVC, %w", err)
```

Prefer this over calling `With()` twice